docs/bok/Activities/License-Management.md

```diff
@@ -0,0 +1,119 @@
+  - Legal Risk
+---
+
+There are several key points that a large enterprise should consider to ensure compliance with open-source license obligations:
+
+1. License identification: The first step is to identify the open-source licenses used in your products and understand the terms and conditions associated with each license.
+2. License compatibility: It is important to ensure that the licenses of the open-source components used in your products are compatible with each other and with your own licenses.
+3. License obligations: Each open-source license has its own obligations, such as the requirement to distribute source code, include a copyright notice, or provide a copy of the license.
+4. Legal review: It is important to have a qualified legal professional review the open-source licenses and obligations to ensure compliance.
+5. Employee education: The enterprise should educate its employees on the importance of compliance with open-source licenses and the obligations associated with each license.
+6. License distribution: The enterprise should ensure that the open-source licenses and relevant notices are included in the distributions of their products.
+
+## License Identification
+
+### For Individual Software Projects
+
+Individual software projects often include direct and transitive dependencies on other pieces of open source software.  There are plenty of tools for cataloguing licenses being used:
+
+1. [Scancode](https://github.com/nexB/scancode-toolkit), recommended here: https://softwarerecs.stackexchange.com/questions/40068/tools-to-scan-code-for-open-source-licenses-and-copyrights
+
+2. [License Checker](https://www.npmjs.com/package/license-checker) which is an NPM module you can use in node projects to list the licenses used by the project.
+
+3. [FINOS' Security Scanning](https://github.com/finos/security-scanning#License-reporting-and-scanning) project provides GitHub Actions which will report licenses on Java (Maven), Node and Python projects via the use of [license-maven-plugin,](https://www.mojohaus.org/license-maven-plugin/) [node-license-validator](https://www.npmjs.com/package/node-license-validator) and [pip-licenses](https://pypi.org/project/pip-licenses/) respectively.
+	
+It is expected that developers would set up these tools to run on the CI/CD pipeline of their projects and generate license reports every time a build is run.
+
+### Across an Estate
+
+Where an organisation is running and managing many projects at once, setting up license scanning on a per-project basis isn't really effective.  Part of the problem is that all the tools listed above have their own output formats.  
+
+Instead, you need tools that will scan open source projects wherever they come from and produce a report in a standard format.  This is called an SBOM.  There are various tools you can use to identify licenses across an estate of projects such as [FOSSA](https://fossa.com), [Mend](https://www.mend.io) and [Snyk](https://snyk.io).
```

Comment:
_This is called an SBOM._ - needs better description of SBOM and/or link to SBOM.md.